index.js

```diff
@@ -76,5 +76,12 @@ export default async function pRetry(input, options) {
 				}
 			}
 		});
+
+		if (options.signal && !options.signal.aborted) {
```

Comment:
`AbortController` and `AbortSignal` are only available on Node 15+. Should we check the Node version here?

Reply:
No, a mention in the docs should be enough.